### ISSUE TITLE
Website loads slowly

### ISSUE THREAD

**Issue by artillect** [HenryDane/EunioaWebsite#7]
This might be partially the fault of my internet, but the website loads astonishingly slow on my computer. I've already made massive strides towards speeding up the loading time (I cut it down by a third last time I checked) but we need to make further improvements to the loading time.

**Comment by HenryDane** [comment]
I agree. Would using nested `gzip`s make things faster (i.e. compress the site as its sent)? I already have it doing it once by default. (I'm not sure how to do more though). Perhaps we could use a script to partially speed up the site by loading consecutively higher res images? Additionally there may be some server-side optimizations to be used as well. 

I agree though, it is slow.

**Comment by artillect** [comment]
I think we could start by further reducing the sizes of images. I’m thinking either a blur to the images in the background or switching the image out for a textured background or a picture of an event would be a good way to handle that. I think gzipping once is already enough as far as compression, so further server-side optimization is probably necessary.

**Comment by HenryDane** [comment]
See #9 -- we are moving hosts. This should fix this.

**Comment by artillect** [comment]
Awesome, glad to see we're finally moving it out of your house haha
